README

```diff
@@ -278,6 +278,15 @@ Features
     - oracle
       Query data from an Oracle database.
 
+    - ovs_events
+      The plugin monitors the link status of OVS connected interfaces,
+      dispatches the values to collectd and send the notification whenever
+      the link state change occurs in OVS DB. It requires YAJL library to be
+      installed.
+      Detailed instructions for installing and setting up Open vSwitch, see
+      OVS documentation.
+      <http://openvswitch.org/support/dist-docs/INSTALL.md.html>
```

Comment:
Yeah, you are right! OvS has changed it (*.md -> *.rst, ) after the PR has been created. Thank you for discovering this :) I will fix it to the right one: http://openvswitch.org/support/dist-docs/INSTALL.rst.html.